### PR TITLE
Add deep map logic to config Aliases

### PR DESCRIPTION
**Pull request by therealmitchconnors**
Without this logic, deep maps are incompatible with aliases.

### CODE REVIEW

File: viper.go [hunk 1]
```diff
@@ -1123,6 +1123,12 @@ func (v *Viper) SetEnvKeyReplacer(r *strings.Replacer) {
 	v.envKeyReplacer = r
 }
 
+// Returns the alias map, used for documentation generation.
+func GetAliases() map[string]string { return v.GetAliases() }
```
Comment (ozevren) [inline viper.go:1127]: Nit: Space between functions.

Also the both needs comments that start with "GetAliases".

File: viper.go [hunk 2]
```diff
@@ -1123,6 +1123,12 @@ func (v *Viper) SetEnvKeyReplacer(r *strings.Replacer) {
...
+func (v *Viper) GetAliases() map[string]string {
+	return v.aliases
```
Comment (ozevren) [inline viper.go]: Nit: Defensively speaking, a copy should be returned, instead of the internal data structure.

File: viper.go (hunk 1 above)
Comment (therealmitchconnors) [reply viper.go:1127]: Normally I would agree, but I'm trying to write this in keeping with the existing style, which comments only the module level functions, and leaves no space between the module function and the instance implementation.  I will add the GetAliases prefix to the existing comment.

File: viper.go (hunk 2 above)
Comment (therealmitchconnors) [reply viper.go]: Good point.  Is there a copy function I should use, or just iterate?

**Review by ozevren — APPROVED** [review]

**Pull request merged**